<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Radar_Settings_Screen</class>
 <widget class="QWidget" name="HNS_Radar_Settings_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LE_RadarSet_MinSpeed">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 MPH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RadarSet_VioSpeed">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Violator Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RadarSet_MinSpeed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Minimum Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Legacy">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>LEGACY
MODE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_RadarSet_Msg4" native="true">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Police">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>POLICE
MODE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_RadarSet_MaxSpeed">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>350</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 MPH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_SpeedMode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Offset
VSL</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RadarSet_TouchMsg">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TOUCH TO SELECT MESSAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Normal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>NORMAL
MODE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_RadarSet_VioSpeed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 MPH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Units">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>MPH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Back">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_RadarSet_Msg2" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RadarSet_MaxSpeed">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>320</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maximum Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RADAR SETTINGS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_RadarSet_Msg1" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Cancel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_RadarSet_Strobes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>STROBES</string>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_RadarSet_Msg3" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RadarSet_Invalid">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>740</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SETTINGS INVALID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>LE_RadarSet_MinSpeed</zorder>
  <zorder>LBL_RadarSet_VioSpeed</zorder>
  <zorder>LBL_RadarSet_MinSpeed</zorder>
  <zorder>PB_RadarSet_Legacy</zorder>
  <zorder>GV_RadarSet_Msg4</zorder>
  <zorder>PB_RadarSet_Police</zorder>
  <zorder>LE_RadarSet_MaxSpeed</zorder>
  <zorder>PB_RadarSet_SpeedMode</zorder>
  <zorder>LBL_RadarSet_TouchMsg</zorder>
  <zorder>PB_RadarSet_Normal</zorder>
  <zorder>LE_RadarSet_VioSpeed</zorder>
  <zorder>PB_RadarSet_Back</zorder>
  <zorder>GV_RadarSet_Msg2</zorder>
  <zorder>LBL_RadarSet_MaxSpeed</zorder>
  <zorder>label_27</zorder>
  <zorder>GV_RadarSet_Msg1</zorder>
  <zorder>PB_RadarSet_Cancel</zorder>
  <zorder>PB_RadarSet_Save</zorder>
  <zorder>PB_RadarSet_Strobes</zorder>
  <zorder>GV_RadarSet_Msg3</zorder>
  <zorder>PB_RadarSet_Units</zorder>
  <zorder>LBL_RadarSet_Invalid</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
